x-pack/plugins/snapshot_restore/public/application/lib/attempt_to_uri_decode.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const attemptToURIDecode = (value: string) => {
+  let result: string;
+  try {
+    result = decodeURIComponent(value);
```

Comment:
I _think_ you might also have to try `decodeURI`. Otherwise, when doing a full page refresh with the details panel open, you might get a 404.

<img width="312" alt="Screen Shot 2020-08-31 at 10 51 43 AM" src="https://user-images.githubusercontent.com/5226211/91733813-15a59f00-eb78-11ea-87c4-80aec37e10f3.png">